gap/isomorph.gi

```diff
@@ -292,23 +284,13 @@ function(D)
 end);
 
 InstallMethod(BlissAutomorphismGroup, "for a digraph and vertex coloring",
-[IsDigraph, IsHomogeneousList],
-function(D, colors)
-  return BLISS_DATA(D,
-                    colors,
-                    fail,
-                    "AutomorphismGroup")[1];
-end);
+[IsDigraph, IsHomogeneousList], {D, colors} -> BLISS_DATA(D, colors, fail)[1]);
```

Comment:
We definitely don't want the last argument in these?

Reply:
Definitely don't, I think this was a remnant of some earlier code.